features/imagebitmap.yml

```diff
@@ -0,0 +1,21 @@
+name: ImageBitmap
+description: The `ImageBitmap` interface represents a bitmap image which can be created from a variety of source objects. It can be drawn to a <canvas> without undue latency, and provides an asynchronous and resource efficient pathway to prepare textures for rendering in WebGL.
+spec: https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#imagebitmap
+status:
+  compute_from: api.ImageBitmap
```

Comment:
While `createImageBitmap` is not baseline, `ImageBitmap` however does appear like it should be.

But neither Caniuse or MDN show it as baseline.

@ddbeck wdyt the right move is here?

Reply:
OK, I misunderstood this question originally.

**The only change we need to make here is to drop the `compute_from` entirely.**

Here's the background:
 
The `createImageBitmap()` method itself _is_ well-supported, but there are some details that that make it look rougher than it is.

On caniuse, Safari is shown as unsupported due to a bug that was fixed over a year ago, relating to the `premultiply` option. Meanwhile, BCD doesn't reflect this bug or the fix. (This needs PRs/issues filed in both places.)

Meanwhile, caniuse says that this depends on the `resizeQuality` option, which Firefox doesn't support. BCD is consistent on this, though I think it's a little bit of a bummer to keep this from being Baseline over it. Oh well.

To recap, there's a number fixes needed for this, but the only one that happens _here_ is that we need to not use `compute_from` here.

Reply:
Opened https://github.com/mdn/browser-compat-data/pull/24496 and https://github.com/Fyrd/caniuse/pull/7160 to capture the premultiply stuff.

Reply:
ack, dropped the `compute_from`, thanks for filing the related issues!